### DISCUSSION TITLE
EddieCon v0.2 (community call + YouTube live  = unconference)

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#7464] [Community Call]
My live stream and community call are being combined to make an exciting event... EddieCon v0.2 Unconference 🚀 

**Please ONLY create a new thread if you would like to submit a talk (first time speakers welcome), but everyone please vote on your favourite talks**

### TALK

Follow these steps if you would like to submit a talk...

1. Visit the [unconference website](https://www.eddiehub.org/eddiecon-02) and fill in the form
2. Start a new thread with your talk details (template below), then the community will upvote their favourite talks (feel free to comment on other threads)

Talk criteria:
- 5mins talk related to Open Source
- 3mins Q&A (don't worry, I will be there to support you)
- Your camera must be enabled as it is a YouTube live stream
- Screen sharing is optional but recommended

*note: unfortunately if there are any technical issues, the speaker will be removed from the stage and we will move on to the next speaker*

### CFP TALK TEMPLATE

Please include the information below in your thread...

```md
##  TALK TITLE

## TALK DESCRIPTION

- What is the talk about?
- What topics will you cover?
```

*note: this discussion will be locked 3 hours before the event*

More information and a video about unconference on the website https://www.eddiehub.org/eddiecon-02

**Comment by Haimantika** [comment]
## Building and thriving an open source community

## TALK DESCRIPTION

- This talk focuses on the essential elements of building and nurturing thriving open-source communities. 
- In this talk we will discuss how we can build and scale an open-source community that is inclusive. Will share tips on how to grow the community and continue the momentum

**Reply by praveenscience** [reply]
Amazing talk! 💪🏻

**Comment by Savio629** [comment]
##  TALK TITLE - Igniting Your Journey: Earning and Building Your Profile through Open Source Projects ⭐

## TALK DESCRIPTION

- What is the talk about?
  This talk is a comprehensive guide on how to kickstart your journey towards earning and building your profile through active contributions to open source projects. It will provide you valuable insights and strategies to increase your chances of getting selected for prestigious events like GSoC, LFX, or any other mentorship programs.
- What topics will you cover?
In this talk I'll try to cover how to maximize your chances of getting selected for mentorship programs like GSoC and LFX. Including tips for selecting the right project, crafting compelling proposals, engaging with the open source community, showcasing your skills, and navigating challenges. Get insights on leveraging mentorship programs and building a strong professional network through this talk.

**Reply by himanshubalani** [reply]
I believe the topic is very good but needs more than 5 minutes to discuss thoroughly and properly. It can't be comprehensive. Several people may have questions but the time is limited. Could you possibly do a seperate livestream on this later as well?

**Reply by Savio629** [reply]
Yeah, the topic is indeed vast and deserves more time for a thorough discussion while I'll try to do my best to provide a valuable overview within the given time frame. Also I'm ready to give a separate session to go into more detail later on...

**Comment by J0SAL** [comment]
##  TALK TITLE - Open Source for Students

## TALK DESCRIPTION

- What is the talk about?
   -  How a newbie/beginner can start with the world of opensource
- What topics will you cover?
   -  Benefits of Joining Communities, Learning in Public, and a list of various OpenSourse Programs available that one can be a part of.

**Comment by Arindam200** [comment]
##  TALK TITLE: Reputation in Tech via Open Source 

## TALK DESCRIPTION

### What is the talk about?
- This Talk is about leveraging open source to enhance tech reputation for Career growth.
### Topics I'll Cover: 
-  Importance of Reputation in Tech Career
- The "SECRET" Formula of Reputation in Tech
- How Does Open Source Help You to grow your Reputation?

**Comment by tobySolutions** [comment]
## How Learning In Public And Creating c
Content Can 10x Your Open Source Journey

## TALK DESCRIPTION

- What is the talk about?
 How someone looking to venture into open source can take advantage of content creation and learning in public to improve and push further in open source generally to attain Maintainer level of big open source projects

- What topics will you cover?

Hacktoberfest, Handling GitHub as a social platform, Using GitHub intentionally, Life of a maintainer.

**Comment by amandamartin-dev** [comment]
##  TALK TITLE:  Inclusivity in Open Source is for everyone

## TALK DESCRIPTION
In this session, you will learn why it is important to lift folks up from underrepresented communities in tech, benefits of intentionally supporting folks from underrepresented communities, and leave with actionable ways to get started today being a tech ally in the Open Source community!

- What is the talk about? How to be a supportive and welcoming member of the open source community and ways to get started.
- What topics will you cover?
What underrepresented groups means
How and why to intentionally support folks in these groups
Some quick ways to get started today

**Reply by amandamartin-dev** [reply]
As I am a speaker as part of my job, please feel free to deprioritize my submission in favor of those new to speaking. Excited to watch everyone.

**Comment by Pradumnasaraf** [comment]
## TALK TITLE: 

**Building an Inclusive Open Source Community: The Role of GitHub Community Standards**

## TALK DESCRIPTION

In this session, you will learn why setting up GitHub Community Standards is important in making the Open Source community inclusive, safe, and welcoming.

- What is the talk about? The talk focuses on GitHub Community Standards, their elements, and how to effectively use them to promote inclusivity.

- What topics will you cover?
   - The importance of GitHub Community Standards for every Open Source project and how to set them up.
   - Components and specific use cases (including LICENSE, PR template, Code of Conduct, Security policy, etc.).
   - How GitHub Community Standards open doors for non-technical individuals to contribute to Open Source.
   - How GitHub Community Standards reduce maintainer time and effort.

**Reply by shubhxg** [reply]
Great idea!!!

**Comment by ChinmayMhatre** [comment]
##  TALK TITLE:  Different Ways of Contributing and Adding Value to an Open Source Project

## TALK DESCRIPTION
This session will help you learn, code and no code contributions that can add value using examples from the community project LinkFree. 

-  What is the talk about? Discussing the do's and don't while contributing, exploring code and no code contributions.

-  What topics will you cover?
   - Navigating through large code bases like LinkFree.
   - Trying to understand, reviewing different PRs.

**Comment by loftwah** [comment]
## Leveraging Open Source and GitHub to Build and Deploy a Ruby on Rails 'Hello World' App to AWS EC2 using GitHub Actions

## TALK DESCRIPTION

My talk is about harnessing the power of open-source tools like Ruby on Rails, GitHub, and AWS to create and manage your software development workflows. Specifically, we'll set up a simple 'Hello World' app in Ruby on Rails, push it to a GitHub repository, and then deploy it automatically to AWS EC2 through GitHub Actions.

The topics I will cover include:

* How to create a new Ruby on Rails project.
* How to push the project to a GitHub repository.
* How to set up a GitHub Actions workflow to automatically deploy the app to AWS EC2 when changes are made to the main branch.

This talk aims to give attendees a basic understanding of leveraging open-source tools to manage their software lifecycle efficiently. It's beginner-friendly but also insightful for those already familiar with these tools.

I look forward to bringing this talk to the EddieCon v0.2 Unconference and diving into the open-source world together!

**Reply by loftwah** [reply]
A backup in case this one is too technical.

**TALK TITLE:** Open-Source DevOps: My Journey and Key Learnings

**TALK DESCRIPTION:**

In this talk, I'll share my journey as a DevOps engineer, how I've leveraged open-source tools and some crucial learnings.

In just five minutes, I'll touch on:

1. My entry into DevOps and how open-source shaped my understanding.
2. An example from my experience utilising open-source DevOps tools (e.g., the Ruby on Rails 'Hello World' app to AWS EC2).
3. A key takeaway from a project where I implemented DevOps principles in an open-source context.

This talk is ideal for those new to DevOps and open-source but will provide some insights for those more familiar with these concepts.

**Reply by ibilalkayy** [reply]
This is nice one. Now you're fully prepared for it.

**Comment by starlightknown** [comment]
##  Automating Documentation Workflows: Streamlining Processes in Open Source Projects

## TALK DESCRIPTION

we will explore the significance of automating documentation workflows to streamline processes in open source projects. This talk aims to shed light on the benefits of automation and how GitHub tools can be leveraged to transform documentation processes, making them more efficient, consistent, and reliable. We will delve into various automation techniques, GitHub tools, and best practices that can be implemented to enhance the quality and accessibility of documentation in open source projects.

Topics include: 
- Introduction to Documentation Workflows in Open Source Projects
- Automation and its Impact on Documentation Workflows
- Leveraging GitHub Tools for Documentation Automation
- Best Practices and Lessons Learned

**Comment by Ananya2001-an** [comment]
##  Mastering the Art of Communication: A Game-Changer in Remote Internships

## TALK DESCRIPTION

- What is the talk about?
   Join me as I delve into the transformative power of effective communication in securing remote work. In this talk, I will share my personal 
   journey and highlight how honing my communication skills became the pivotal factor in unlocking remote opportunities. 

- What topics will you cover?
   In this talk, I'll share valuable insights on approaching open source projects and effectively communicating with project maintainers. I'll 
   guide you on making meaningful contributions that enhance their experience, rather than creating challenges. Whether you're new to open 
   source or seeking to generate passive income, this beginner-friendly session will equip you with essential tips and strategies for success.

   I am still a newbie in the world of open source and there's a lot of room for learning but I hope with this talk I can help other open source 
   enthusiasts somehow 😄 :octocat:

**Comment by aasthashahcodes** [comment]
## Talk Title: Journey of a non-tech individual into Tech

## Talk Description: 

I'll be talking about how I transitioned my career into tech and what helped me grow. Importance of Open Source & communities in my growth.

Topics:
1. How to commence your journey in tech.
2. Contribution to open-source communities
3. Mistakes to avoid as a beginner

**Comment by shoebilyas123** [comment]
## TALK TITLE - History and Future of Open Source

## TALK DESCRIPTION
# What is the talk about?
 -  This talk will be about a general timeline of open source development and how it started. The talk will also cover what the future of open source looks like 

# What topics will you cover?
- Introduction to open source development.
- First open source project and how it started.
- A general timeline.
- Future of open source development.

**Comment by Nishu0** [comment]
##  TALK TITLE - Unleashing the Potential of Open Source Tools in Hackathons

## TALK DESCRIPTION

- What is the talk about?
This talk is dedicated to exploring the symbiotic relationship between hackathons and open source tools. I will delve into the transformative power of open source software in fostering innovation, collaboration, and rapid prototyping during hackathons.

- What topics will you cover?
**1. Introduction to Hackathons:**
Definition and purpose of hackathons.
Emphasis on the time-constrained, collaborative nature of hackathon events.
**2. The Role of Open Source Tools in Hackathons:**
Explanation of open source tools and their relevance in hackathon settings.
Discuss how open source tools provide a solid foundation for rapid prototyping, iteration, and experimentation.
**3. Popular Open Source Tools for Hackathons** 
Examples: [ngrok](http://ngrok.com/), [graphicburger](http://graphicburger.com/), [codepen](http://codepen.io/), [codyhouse](http://codyhouse.co/)

**Comment by Shivam-Katare** [comment]
# Talk Title - Open Source: Unlocking Opportunities in Remote Work

# Talk Description

## What is the talk about?

This talk explores the collaborative impact between open-source software and remote work, highlighting the opportunities and benefits they offer. It covers the rise of remote collaboration, showcases open-source projects driving remote work opportunities, and provides insights for building a remote career using open source.

## What topics will you cover?

- The rise of remote work and its impact on the job market.
- Remote collaboration with open source: tools, platforms, and workflows.
- Showcase of open source projects driving remote work opportunities.
- Building a  remote career with open source: skills and experiences.
- Resources for finding remote jobs and contributing to open source.

**Reply by mathcodes** [reply]
I'd love to hear what you've got to say about these topics!

**Comment by DhruviThakkar210** [comment]
TALK TITLE : 19 Y/O FOUNDER WHO MADE HER FIRST $703  THROUGH OPEN SOURCE ! 

TALK ABOUT : 
HOW I STARTED A GLOBAL COMMUNITY FOR WOMEN'S IN TECH  AND HOW I MADE $703 ? 


TOPICS I WILL COVER  : 

👁️  HOW  TO THROW BID 
 👁️ HOW OPEN SOURCE CAN DO WONDERS ... ESPECIALLY FOR WOMEN'S IN TECH.
👁️ HOW I AM BUILDING GLOBAL COMMUNITY

**Reply by denabyet** [reply]
You are really inspiring 💪

**Comment by GrahamTheDevRel** [comment]
##  TALK TITLE
How to make your OSS Project ReadMe More Accessible.

## TALK DESCRIPTION
There are 1 **Billion** people in the world with a disability.

This talk will help make sure that your Readme Markdown is properly structured and accessible for as many people as possible.

Essentially: The do's and don'ts when creating your project's Readme.md file, covering things like:
- using tables for layouts 👎🏼 
- how to write good alt tags on images 🤩, 
- proper heading structure 🚀 
- pictures of code ❌

It is amazing how in just 5 minutes you can make sure your Markdown is properly structured and accessible to everyone.

**Reply by mathcodes** [reply]
Love this Graham. Easy implementation for those in-need yet way overlooked. 🔥

**Comment by ibilalkayy** [comment]
# **TALK TITLE**

**Navigating the Open Source Landscape: A Journey from Novice to Contributor.** 

# **TALK DESCRIPTION**

The talk will be divided into five key sections:

## **Introduction (1 min):**

I will start by introducing myself and sharing my background, with a special emphasis on my experience in open-source contribution. I will explain why I decided to contribute to open-source projects and what my initial expectations were.

## **The Open Source World (1 min):**

Here, I will explain what open source is and its significance in today's software development landscape. I will also briefly discuss some of the major open-source projects that exist and how they are impacting the tech industry.

## **My First Steps (1 min):**

In this section, I will share the story of my first venture into open-source contribution. I will talk about how I chose the projects to contribute to, the challenges I faced when making my first contributions, and how I overcame them. This will involve sharing some key learnings for beginners in the open-source space.

## **The Journey So Far (1 min):**

I will discuss my journey as a contributor to Linux for Pirates. I will share some of my significant contributions and the impact they had on the projects. This section will also highlight the skills I developed, the relationships I formed, and the opportunities I encountered along the way.

## **Conclusion and Q&A (1 min):**

I will conclude the talk by summarizing the key takeaways and opening the floor for a Q&A session.

Key Takeaways:

- Understanding the significance of open source contribution in today's tech industry
- A step-by-step guide for newbies on how to start contributing to open-source projects
- Real-life experiences, challenges, and learnings from a seasoned open-source contributor.

Through this talk, I aim to inspire more software developers to embark on their journey of open-source contribution and demonstrate how they can leverage it for their professional growth.

**Comment by suitupstan** [comment]
# The Art Of Creating Amazing Open Source Projects

## TALK DESCRIPTION

- What is the talk about?

There are millions of repositories on github, to stand out to users you need to have a great community, one which comes from contributors, and attracting contributors is hard. I wanna cover some best practices that will help you make your better for contributors :D

Are you a contributor? You can still checkout this session to learn which projects are gonna be a great experience to contribute to!

- What topics will you cover?

What should be in a readme, how should you form your issues, usage of labels and automation, and a short description on how my project `gardevoir` reached 1k stars

**Reply by paharipratyush** [reply]
Nice Talk Idea

**Reply by shubhxg** [reply]
Great Idea

**Reply by harshcodesdev** [reply]
Sounds good 👍

**Reply by suitupstan** [reply]
Thanks Pratyush, Shubh & Harsh!

**Reply by manogna-chinta** [reply]
@gptkrsh nice one

**Reply by suitupstan** [reply]
Thanks @Manogna-chinta!

**Comment by hemantwasthere** [comment]
#  Open Source: Driving Social Impact and Community Empowerment

## TALK DESCRIPTION

### What is the talk about?
- How open source initiatives have positively transformed society, fostered collaboration, and empowered individuals and organizations to make a difference

### What topics will you cover?
- The Power of Open Collaboration
- Accessibility and Affordability
- Local Empowerment and Capacity Building
- Social Innovation and Customization
- Creating Sustainable Solutions

**Comment by shivangshandilya** [comment]
# Do's and Don'ts when Contributing to Open Source

## Talk Description

### <b>What is the talk about?</b>
- You know how to contribute to Open Source projects but do you know what are the important Do's and Don'ts for doing the same that you should keep in mind? Well, this talk will highlight some practices that will make your contributions even better!!

### <b>What topics will you cover? </b>
- My Introduction
- What Open Souce means to me
- Do's and Don'ts while contributing

**Reply by SagarSharma2809** [reply]
I like this topic , its simple and most beginner doubts will be covered

**Reply by satyampsoni** [reply]
Hope this will clear the doubts of beginners about what kind of mistakes to be avoided.

**Reply by PriyanshNegi** [reply]
All the best!

**Reply by sarthaksarthak9** [reply]
I think this will really help newbie's. All the very best !!

**Reply by shivangshandilya** [reply]
> I think this will really help newbie's. All the very best !!

Thanks, Sarthak 😁

**Reply by paharipratyush** [reply]
Great topic, Will be helpful for all the beginners in Open Source

**Comment by vedanshiprajapati** [comment]
# Intro to Open-Source Security

## TALK DESCRIPTION

- ###  what is the talk about?
   this talk will be about giving introduction to open-source security and  which one is more better between closed source and open- 
   source. it will provide basic information about the tools and software that are used to secure an open-source work.
- ### what topics i will cover?
   introduction to OSS
   why software companies are preferring open-source now?
   software which are used for OSS

**Comment by SaarthakMaini** [comment]
##  Making Open Source Contributions Like A Pro ! 🌟 

## TALK DESCRIPTION

- What is the talk about?

Are you stuck on just solving Good First Issues and don't know how to move forward? Would you like to start making bigger, better and more complex contributions like a pro? Well in this session we're going to discuss exactly how YOU can increase your level and take your contribution capabilities up a notch!

- What topics will I cover?
  - Introduction
  - Tips To Start Making Big and Complex Open Source Contributions
  - Tips To Understand Large Codebases 
  - Where The Audience Could Connect With Me For Asking Further Doubts

**Reply by Ritwik1607** [reply]
Interesting topic.
Eagerly waiting for this session.

**Reply by SaarthakMaini** [reply]
Thank you @Ritwik1607 ! 😄

**Reply by 3xp11i** [reply]
Good to see you are participating!

**Reply by SaarthakMaini** [reply]
Thank you @prakhartiwari0 ! Thank you for letting me know about the event !😄

**Reply by SuskeDevs** [reply]
Good luck Saarthak!

**Reply by SaarthakMaini** [reply]
Thank you @Keshavshyper6325 !😁

**Reply by tarun-pratap** [reply]
Good luck

**Reply by SaarthakMaini** [reply]
Thank you very much @tarun-pratap !😁

**Comment by anmolkhetan** [comment]
ALL THE BEST

**Comment by karthiknadar1204** [comment]
<h1> Open source for new contributors </h1>

Talk Description:
- Open source codebases can be overwhelming to someone who tries to venture into the space for the first time. As someone who has been contributing for over the past few months, i wish to share my insights and learnings on how not to get overwhelmed by the git commands and the large codebases and role of community in building up my confidence and encouraging me to move forward and my views on open source to become the new way for hiring people.


Topics to cover
- How can someone just getting started out with open source not be overwhelmed by the plethora of commands and steps to be followed and the large codebases before making their first contribution .
- How can a complete beginner get involved in open source communities and how does community support and empower you to achieve more.
-  My views on open source as the new way to showcase your skills and get hired.

**Reply by RaghavKumarSagar** [reply]
All the best bro !!!

**Comment by nitishfy** [comment]
##  Stop Looking For Good First Issues in Open Source: 3 Untold Lessons No One Teaches You

## TALK DESCRIPTION
How many times you’ve heard someone saying “If you’re getting started in Open Source, good-first issues are the best fit to get started with”? While this can be true for some projects, this is not always the correct way to get started. With the growing demand for Open Source software, new folks leave no stone unturned to put forward significant contributions to Open Source projects even in minute ways.

In this talk, Nitish Kumar who has been an active contributor to various Open Source projects such as Kubernetes, docker, and so on will be presenting the five HARSH and UNTOLD lessons from the Open Source community. From having zero experience on how to articulate ideas effectively, and being unable to contribute the bare minimum, Nitish has come a long way.

He will also be sharing the ideal approach to deal with various common misconceptions that contributors tend to assume - The community is ALWAYS there to support you, Docs are pretty simple to get started with, and whatnot. But all this is not true! Whether you’re a beginner finding your own way to get started or someone who is contributing to a project but not able to make up significant contributions, this session will change your perspective on how to look at the community, present your ideas well and contribute effectively.

**Reply by shivangshandilya** [reply]
Would love to hear about this 🔥

**Comment by mathcodes** [comment]
##  AI & Open Source: New Guidelines and Implications for Open Source Projects

## TALK DESCRIPTION
In an age of increasing reliance on AI technologies, websites and open-source projects are facing new challenges and opportunities. This talk is designed to shed light on the significance of emerging AI guidelines for websites, highlighting how they impact the dynamic open-source environment.

 - What is the talk about?

The talk will start by unpacking the recent AI guidelines for websites and their primary intentions. It will explain the guidelines, focusing on aspects such as user privacy, data security, user interface, content curation, and more. It will also cover how AI's growing involvement necessitates these guidelines and the potential repercussions of non-compliance.

After establishing a comprehensive understanding of the guidelines, the talk will explore the specific implications these regulations have on open-source projects. It will analyze how open-source contributors can incorporate these guidelines into their projects and what changes this might bring about.


- What topics will you cover?

Overview of the new AI guidelines for websites: their origin, intention, and areas of focus.
The necessity for AI guidelines: exploring cases of misuse and repercussions.
Impacts of these guidelines on website design, functionality, and user experience.
Intersection of AI guidelines and open-source projects: changes, challenges, and opportunities.
Strategies for open-source contributors to adopt and adapt to these guidelines effectively.
Through this talk, the audience will gain insights into the evolving landscape of AI application in websites and how open-source projects can proactively adapt to these changes, ensuring the longevity and relevance of their contributions.

**Reply by shivangshandilya** [reply]
Interesting topic, would definitely love to hear your views on this.

**Reply by mathcodes** [reply]
Thanks, @ShivangShandilya. Whether tomorrow or on back channels, I'll be around 👍

**Comment by avionmission** [comment]
##  Hackathons, Open-source & your Portfolio

## TALK DESCRIPTION

What is the talk about?
- A 5-minute talk on how participating in hackathons can lead to creating great open-source projects in which you can build a community around, and help you learn so much.

What topics will you cover?
- My Experiences with Hackathons.
- Why you should make your hackathon projects OPEN-SOURCE!
- Why GitHub is the best portfolio for a developer.

**Reply by yuvicc** [reply]
Nice title🔥

**Comment by yuvicc** [comment]
##  The Dark Side of Open Source

## TALK DESCRIPTION
"Oh! Please don't get confused with the title, Open Source is the best thing that has happened to me." I can see the hype around Open Source on Twitter, but I don't see many people talking about it. That's why I've decided to speak up and initiate a discussion about it!

### What is the talk about?
- How does it feel to be an OS Maintainer?
Here we will discuss some of the challenges that maintainer faces in their daily life. 

- What is the significant difference between an author and a contributor?
In our discussion, we will explore the perspectives of OS contributors and maintainers while contributing to a project. It's not the same as doing voluntary work in a park and being the park keeper.

- Money in Open Source
We will also discuss how financial support plays an important role in the life of OS maintainer. OS libraries/framework was never about financial gain or money making. 

- Supporting the OS maintainer
At last we will be discussing how to support the maintainers and make a healthy balance in the OS ecosystem.

**Reply by avionmission** [reply]
Very interesting! Looking forward to listening to this talk 🤝
